Messages/MessageTypes.h

```diff
@@ -15,5 +15,6 @@
 
 
 enum class MessageType {
```

Comment:
Could you add a quick description of the purpose of this class and how to fill it in the future ? I understood this is where you store the message types, but it's not obvious.

I didn't understand either why is there DataRequest = 0 inside the message types.